### ISSUE TITLE
Set boundary for calculating p-value in fgseaMultilevel

### ISSUE THREAD

**Issue by niehues** [yigbt/multiGSEA#9]
Hi, 
Thanks for developing this R package.
I'm running `multiGSEA::multiGSEA` with a small metabolomics data set - it takes quite a long time and uses a lot of memory. I stopped the execution and tried running `fgsea::fgseaMultilevel` called by `multiGSEA` directly. This works fine with the `fgsea` default value (`eps = 1e-10`) for the `eps` parameter (boundary for calculating the p-value) but takes very long for `eps = 0`.

It would be useful to be able to change the `eps` parameter when running `multiGSEA::multiGSEA`. 
Best,
Anna

**Comment by boll3** [comment]
Hi Anna,

unfortunately, I wasn't able to reproduce your runtime and memory issues.
Of course, the p-value calculation takes more time when the cutoff is set to 0.
But in my case, it still finished in a reasonable time compared to `eps = 0` even though I used several thousand pathway definitions.

However, you are still correct, that it might be useful to implement `eps` as a function parameter.
I updated the devel version and the current release 3.16 and the changes should be visible in the next 48 hours!

Thanks for your suggestion!

Cheers,
Sebastian
